components/omega/src/ocn/Tracers.cpp

```diff
@@ -524,4 +515,30 @@ I4 Tracers::updateTimeLevels() {
    return 0;
 }
 
+//---------------------------------------------------------------------------
+// get index for time level
+// TimeLevel == [1:new, 0:current, -1:previous, -2:two times ago, ...]
+//---------------------------------------------------------------------------
+I4 Tracers::getTimeIndex(I4 &TimeIndex, const I4 TimeLevel) {
```

Comment:
Since the Tracers module follows the time level convention of OceanState, this function can be identical to the getTimeIndex function of OceanState.

```suggestion
I4 Tracers::getTimeIndex(I4 &TimeIndex, const I4 TimeLevel) {

   // Check if time level is valid
   if (NTimeLevels > 1 && (TimeLevel > 1 || (TimeLevel + NTimeLevels) <= 1)) {
      LOG_ERROR("Tracers: Time level {} is out of range for NTimeLevels {}",
                TimeLevel, NTimeLevels);
      return -1;
   }

   TimeIndex = (TimeLevel + CurTimeIndex + NTimeLevels) % NTimeLevels;

   return 0;
}
```

Reply:
I have updated both of these functions, thanks.